### ISSUE TITLE
highway=steps should be treated as a footway, not a road

### ISSUE THREAD

**Issue by AntiCompositeNumber** [tsmock/sidewalks#15]
![image](https://github.com/tsmock/sidewalks/assets/10331574/d526421e-0ae9-4b95-8d71-d385b856f5ef)

It doesn't make a lot of sense to add kerbs and a crossing for an intersection with `highway=steps`.

**Comment by tsmock** [comment]
Can I have a sample area? I'd like to make certain I understand the problem.

I'm envisioning a crossing where the sidewalk comes down a hill via steps, where the steps terminate _at the roadway_ and not at another sidewalk.

I'll have to check and see what is supposed to happen in those cases. Example: if there is a raised edge at the edge of the roadway, does the first step of the stairway count as a kerb?

**Comment by AntiCompositeNumber** [comment]
The steps I was mapping near were already mapped and were connected to the road way on OSM, but physically stop at the sidewalk. As I mapped the sidewalk, I placed nodes close enough to the steps way that the plugin mapped it as a crossing. I then had to remove the crossing kerbs and split in addition to cutting the steps back. The specific set of steps was https://www.openstreetmap.org/way/1225561899, if that's helpful.
